spec/02-integration/04-admin_api/04-plugins_routes_spec.lua

```diff
@@ -442,6 +442,7 @@ for _, strategy in helpers.each_strategy() do
             local body = assert.res_status(200, res)
             local json = cjson.decode(body)
             assert.is_table(json.fields)
+            assert.is_table(json.entity_checks)
```

Comment:
Please assert the content of the new entry.
Please add a negative test to make sure that 'entity_checks' JSON key is not present when it is absent from the Lua schema.

Reply:
I'd argue that we always return `entity_checks` in the JSON response even when the value is empty for the sake of consistency.